### PR TITLE
LOOC has a prefix now

### LINKED ISSUE #8833: LOOC doesn't have a prefix
## Description
LOOC should have a prefix

### PR DESCRIPTION
**Pull request by Just-a-Unity-Dev**
## About the PR 
resolves #8833, the LOOC prefix is `|`

**Screenshots**

https://user-images.githubusercontent.com/67359748/173511365-70fe0068-4047-4f3e-89de-3d9545b849bd.mp4

**Changelog**

:cl: eclips_e
- add: Added a prefix for LOOC, the prefix is now |

### CODE REVIEW

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Swap it with admin chat as it barely gets used and makes much more sense.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Content.Client/Chat/UI/ChatBox.xaml.cs [hunk 1]
```diff
@@ -61,7 +61,7 @@ public partial class ChatBox : Control
         public const char AliasDead = '\\';
         public const char AliasOOC = '[';
         public const char AliasEmotes = '@';
-        public const char AliasAdmin = ']';
+        public const char AliasLOOC = ']';
```
[the remark this answers is not in the document]
Comment (Just-a-Unity-Dev) [reply Content.Client/Chat/UI/ChatBox.xaml.cs]: i have commited a sin. a webedit + mobile. reviews addressed

**Comment by metalgearsloth** [comment]
This might conflict with hud refactor considering they refactored chat so gonna sit until that's merged.

**Comment by wrexbe** [comment]
I looked at the UI refactor a bit, this code isn't going to be compatible. It looks like LOOC will have a prefix of '(' when it's done.

**Pull request closed without merging**